logic/src/commonMain/kotlin/com/wire/kalium/logic/data/client/remote/ClientRemoteDataSourceImpl.kt

```diff
@@ -15,7 +16,13 @@ import com.wire.kalium.network.utils.isSuccessful
 class ClientRemoteDataSourceImpl(
     private val clientApi: ClientApi,
     private val clientMapper: ClientMapper
-): ClientRemoteDataSource {
+) : ClientRemoteDataSource {
+
+    private companion object {
```

Comment:
may the companion object be  moved to the end of the class
[Class Layout Formatting](https://wearezeta.atlassian.net/wiki/spaces/AR/pages/122520039/Code+Style+Guideline#Positioning-of-elements)

Reply:
Thanks for the reference, fixed the order new commit.